projects/core/src/lib/mask.ts

```diff
@@ -78,13 +74,7 @@ export class Maskito {
             return;
         }
 
-        const maskModel = new MaskModel(this.elementState, this.options);
-
-        try {
-            maskModel.addCharacters(this.elementState.selection, pressedKey);
-        } catch {
-            event.preventDefault();
-        }
+        return this.handleInsert(event, pressedKey);
```

Comment:
`return` seems to be unnecessary here

Reply:
https://github.com/Tinkoff/maskito/pull/35/commits/de0c87bead9dd257c0e83dd0c294db573da115f9